app/models/game.rb

```diff
@@ -3,4 +3,48 @@ class Game < ApplicationRecord
   belongs_to :opponent, class_name: 'User', foreign_key: 'opponent_id'
 
   scope :available, -> { where(Game.arel_table[:owner].not_eq(0)).where(opponent = nil) }
+
+  def populate_game!(params.require(:piece).permit(:type, :position_x, :position_y, :game_id, :color, :captured))
+    Piece.create("Rook", 0, 0, @game.id, "white", false)
+    Piece.create("Knight", 1, 0, @game.id, "white", false)
+    Piece.create("Bishop", 2, 0, @game.id, "white", false)
+    Piece.create("King", 3, 0, @game.id, "white", false)
+    Piece.create("Queen", 4, 0, @game.id, "white", false)
+    Piece.create("Bishop", 5, 0, @game.id, "white", false)
+    Piece.create("Knight", 6, 0, @game.id, "white", false)
+    Piece.create("Rook", 7, 0, @game.id, "white", false)
+    Piece.create("Pawn", 0, 1, @game.id, "white", false)
```

Comment:
@LampshadeWarrior is there a way we can create pawns in a shorter way? Just for the sake of exercising some ruby.. all the pawn lines seem to be very similar.. maybe using a range like here: http://ruby-doc.org/core-2.2.0/Range.html#method-i-each

Reply:
@ilyakrasnov I just refactored my code and pushed it!